### PR TITLE
add OIDC auth plugin

### LINKED ISSUE #96: No Auth Provider found for name "oidc"
I am getting this error `ERRO[0000] can't get Kubernetes client: No Auth Provider found for name "oidc"` . Does the cluster have to have OIDC RBAC?

### PR DESCRIPTION
**Pull request by prydonius**
fixes #96

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -40,8 +40,9 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 
-	// Adding explicitely the GCP auth plugin
+	// Explicitly adds the GCP and OIDC auth plugins
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```
Comment (andresmgot) [inline cmd/root.go]: probably it is better to do:
```
_ "k8s.io/client-go/plugin/pkg/client/auth
```
so that include the plugin for Azure as well

Comment (prydonius) [reply cmd/root.go]: oh, you're right - didn't see that the top-level package includes them all

**Comment by prydonius** [comment]
@andresmgot PTAL, had to update deps

**Comment by andresmgot** [comment]
LGTM

**Pull request merged**